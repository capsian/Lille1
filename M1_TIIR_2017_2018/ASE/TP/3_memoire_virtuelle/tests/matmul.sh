#!/usr/bin/env bash

echo "=== MATMUL ==="
../build/bin/mmu_manager > matmul.out

echo "=== ORACLE ==="
../build/bin/oracle < matmul.out
echo ""