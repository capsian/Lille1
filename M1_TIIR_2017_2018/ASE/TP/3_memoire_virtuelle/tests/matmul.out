[Starting user process]
matrices size: 400x400
vm used:   468 pages
pm space:   256 pages
vm space:  4096 pages
initializing matrices
multiplying matrices ...............